docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-NP-complete-vs-NP-hard-con.adoc

```diff
@@ -0,0 +1,26 @@
+[id='optimizer-NP-complete-vs-NP-hard-con']
+= A planning problem is NP-complete or NP-hard
+
+The provided use cases are _probably_ http://en.wikipedia.org/wiki/NP-complete[NP-complete] or harder.
+NP-complete means:
```

Comment:
The definition of NP-complete and NP-hard are a pain, because as long as "P vs NP" (see wikipedia) isn't settled, it's hard to phrase it. Anyway, that being said, I would rephase this as:

The provided use cases are _probably_ https://en.wikipedia.org/wiki/NP-completeness[NP-complete/NP-hard],
which means:

Reply:
I've made this: 

"The provided use cases are _probably_ http://en.wikipedia.org/wiki/NP-complete[NP-complete or NP-hard], which means the following statements apply:" 

purely to conform with IBM style guide restrictions on introducing lists with incomplete sentences.